src/Administration/Resources/app/administration/src/module/sw-settings-rule/snippet/de.json

```
@@ -98,6 +98,11 @@
       "messageSaveSuccess": "Die Regel \"{name}\" wurde erfolgreich gespeichert.",
       "ruleCard": "Bedingungen",
       "messageSaveError": "Beim Speichern der Regel ist ein Fehler aufgetreten",
+      "productStreamIndexingWarning": {
+        "title": "Indexierung dynamischer Produktgruppen ist deaktiviert",
+        "message": "Die Regelbedingung „Position in dynamischer Produktgruppe“ wird immer als falsch bewertet.",
```

as this is a message for merchants, maybe something like that would be a better message: 

```suggestion
        "message": "Die Regelbedingung „Position in dynamischer Produktgruppe“ trifft daher nie zu. Aktivieren Sie die Indexierung, um diese Bedingung verwenden zu können.",
```

---

@GitEvil: what do you think?

---

FYI, I adapted the text from [this comment in the other ticket](https://github.com/shopware/shopware/issues/11142#issuecomment-3149124426)

---

yes, its OK. The wording could still be better I think. @fabianhueske what do you think about the suggestion?

---

Hi 😊 Good catch. Let's use:

> The condition “Item in dynamic product group” will never be true.

and
> Die Bedingung „Position in dynamischer Produktgruppe” wird nie zutreffen.

I've updated the comment on the other issue as well. Since we're already in the Rule Builder at Conditions, we can skip the context and jump straight to the wording 'Condition'. I like your suggestion, @ennasus4sun, of providing instructions on how to solve this 'problem', but I don't think they are necessary and we can save some space by leaving them out. The headline and description should provide users with enough information to understand that something is disabled, therefore not working, and that it will work again when it is enabled.